pkg/componentreadiness/resolvedissues/types.go

```diff
@@ -0,0 +1,240 @@
+package resolvedissues
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/openshift/sippy/pkg/apis/api"
+	"github.com/openshift/sippy/pkg/util/sets"
+)
+
+type ResolvedIssue struct {
+	TestID   string
+	TestName string
+	Variant  api.ComponentReportColumnIdentification
+
+	Issue Issue
+
+	ImpactedJobRuns []JobRun
+}
+
+type Issue struct {
+	IssueType IssueType
+
+	Infrastructure *InfrastructureIssue
+	PayloadBug     *PayloadIssue
+}
+
+type InfrastructureIssue struct {
+	// required
+	Description string
+	// optional
+	JiraURL string
+	// TODO consider whether jira URL should be required and get the resolution date from there
+	ResolutionDate time.Time
+}
+
+type PayloadIssue struct {
+	// required
+	PullRequestURL string
+
+	// TODO switch to detecting this from the payload
+	// required
```

Comment:
The payload approach is tricky, you only get the time it was created IIRC.  We probably need to start storing the time it was accepted.  Until then you'll have to figure the payload time + some number of hours.  There's a delay from payload creation to job start, and then the duration of the running jobs.  8 hours?

There are cron jobs running as well that get pulled into CR, which could start after the nightly was created but be running the old payload.

Reply:
> but we only have the creation time; add job start delay and duration (around 8 hours?).

I don't understand what action I should take in this PR based on this.

Reply:
I'm commenting on a TODO so nothing if you don't plan to do this now

Reply:
>I'm commenting on a TODO so nothing if you don't plan to do this now

ok, thanks.  I don't plan to do anything about this now.